nx/source/services/set.c

```diff
@@ -348,6 +348,76 @@ Result setsysGetColorSetId(ColorSetId* out)
 
 }
 
+Result setsysGetSettingsItemValue(const char *name, const char *item_key, void *value_out, size_t value_out_size) {
+    IpcCommand c;
+    ipcInitialize(&c);
+    ipcAddSendStatic(&c, name, SET_MAX_NAME_SIZE, 0);
```

Comment:
Does official sw use fixed SET_MAX_NAME_SIZE or the actual stringlen?

Reply:
Official software does use this, however it was meant to be 0x48 not 0x30, since it seems the Pegaswitch script I was referencing was wrong.